cmake/Findwebp.cmake

```
@@ -28,8 +28,22 @@ set(webpdemux_LINK_LIBRARIES "" CACHE STRING "Extra link libraries of webpdemux"
 
 set(webpdemux_LINK_FLAGS "" CACHE STRING "Extra link flags of webpdemux")
 
+find_library(libwebpmux_LIBRARY
+    NAMES libwebpmux
```

Remove the `lib` prefix: this would be looking for `liblibwebpmux.so` otherwise.
```suggestion
    NAMES webpmux
```